net-im/telegram-desktop-bin/telegram-desktop-bin-1.7.14.ebuild

```diff
@@ -51,11 +51,6 @@ pkg_preinst() {
 
 pkg_postinst() {
```

Comment:
Hm, isn't those two phase redeclarations redundant now?

Reply:
In case the xdg stuff is redundant, is it?

Reply:
I mean, xdg is the only eclass providing postinst/postrm inherited, so you don't need to restate those phases explicitly.

Reply:
done